base/applications/taskmgr/taskmgr.c

```diff
@@ -482,14 +482,23 @@ TaskManagerWndProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam)
         TrayIcon_UpdateIcon();
         break;
 
+    case WM_INITMENUPOPUP:
+        /* Do not disable the status bar if we opened the system menu */
+        if (!HIWORD(lParam))
+            TaskManager_DisableStatusBar(hDlg);
+        else
+            TaskManager_EnableStatusBar(hDlg);
```

Comment:
Shouldn't the re-enabling be done in an `WM_UNINITMENUPOPUP`? Or maybe the call done in `WM_EXITMENULOOP` is sufficient?

Reply:
WM_EXITMENULOOP is sufficient

Reply:
If you're talking about the `else` clause then see one of the comments above

Reply:
Yes, it does happen on Windows too

Reply:
OK actually this also handles "smoothly" the transition between the "user" menu and the system one (and back).